usagov/vote-gov-drupal
# VOTE-2182 update global metatag sharing image

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2182

## Description

Update global sharing image.

## Deployment and testing

### Post-deploy steps

1. run `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/ and confirm the social sharing image is updated in the source of the page

## Checklist for the Developer




## Checklist for the Peer Reviewers
